Palindrome.java

```diff
@@ -0,0 +1,18 @@
+public boolean isPalindromeRecursive(String text){
+    String clean = text.replaceAll("\\s+", "").toLowerCase();
+    return recursivePalindrome(clean,0,clean.length()-1);
+}
+
+private boolean recursivePalindrome(String text, int forward, int backward) {
```

Comment:
can you comment on the mechanism here?

Reply:
Sorry, I am a newbie. This method has three variables?

Reply:
Public keyword makes it such that the methods can be accessed outside the given class